hack/generate.sh

```diff
@@ -36,7 +36,9 @@ function generate_python_client() {
         -e SWAGGER_FILE=/swagger.yaml -e OUTPUT=/local/assisted-service-client/ \
         quay.io/ocpmetal/swagger-codegen-cli:2.4.15 /script.sh
      cd "${dest}"/assisted-service-client/ && python3 setup.py sdist --dist-dir "${dest}"
-     cd "${dest}"/assisted-service-client/ && python3 "${__root}"/tools/client_package_initializer.py "${dest}"/assisted-service-client/  https://github.com/openshift/assisted-service --build
+     cd "${dest}"/assisted-service-client/ && python3 "${__root}"/tools/client_package_initializer.py "${__root}"/tools/python_api_client_extra_files/ "${dest}"/assisted-service-client/  https://github.com/openshift/assisted-service --build
+     rm "${dest}"/assisted-service-client-1.0.0.tar.gz
+     cp "${dest}"/assisted-service-client/dist/assisted-service-client-*.tar.gz "${dest}"/
```

Comment:
can you explain what happens here?

Reply:
1. Updating the client and generating wheel and tar.gz artifacts
2. Remove the old artifact that was created so far
3. Copy the new source artifact to where the previous one was